examples/text_summarization/bertsum_cnndm.ipynb

```diff
@@ -0,0 +1,3921 @@
+{
```

Comment:
<p>I thought we already did both prediction and evaluation in the cells above. What's done in the cells below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>